File: php/http_request/usercode.php
<?php

require '/libs/vendor/autoload.php';
use GuzzleHttp\Client;

function handle($data) {

    $client = new Client();

    $response = $client->request('GET', 'https://reqres.in/api/users?page=1');
    
    $data['res'] = json_decode($response->getBody(), true);

    return $data;
}

File: python/http_request/usercode.py
import urllib.request
import json

def handle(data):
    req = urllib.request.Request("https://reqres.in/api/users?page=1", method="GET")
    req.add_header("User-Agent", "urllib-example/0.1")

    f = urllib.request.urlopen(req, timeout=3)

    data["res"] = {
        "code": f.getcode(),
        "body": json.loads(f.read()),
    }

    return data
